src/components/AvatarGroup/AvatarGroupCounterTooltipContentVirtualizedList.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { CSSProperties, ReactElement, Ref, useMemo } from "react";
+import VirtualizedList from "../VirtualizedList/VirtualizedList";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { AvatarType } from "../Avatar/AvatarConstants";
+import { AvatarProps } from "../Avatar/Avatar";
+import styles from "./AvatarGroupCounterTooltipContentVirtualizedList.module.scss";
+
+const LIST_OPTIONS = Object.freeze({
+  maxItemsWithoutScroll: 10,
+  itemLineHeight: 34,
+  itemLineWidth: 150
+});
+
+interface AvatarGroupCounterTooltipContentVirtualizedListProps extends VibeComponentProps {
+  /**
+   * Array of Avatar components
+   */
+  avatarItems?: { value: AvatarProps & { tooltipContent: string | ReactElement } }[];
```

Comment:
I see that the type `{ tooltipContent: string | ReactElement }` (and perhaps even `{ value: AvatarProps & { tooltipContent: string | ReactElement } }`) is repeating , maybe extract the type?